app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -198,7 +198,8 @@ def center_toolbar_filename_services
         "services_center_tb"
       end
     elsif x_active_tree == :ot_tree
-      if %w[root xx-otcfn xx-othot xx-otazu xx-otazs xx-otvnf xx-otvap].include?(x_node)
+      return nil if x_node == "xx-otovf" || @record.kind_of?(ManageIQ::Providers::Vmware::InfraManager::OrchestrationTemplate)
```

Comment:
Ugh :confused: can't we do this somehow nicer?

Reply:
> Ugh confused can't we do this somehow nicer?

@skateman i can control the code to disable all buttons on summary screen for OVF templates, but could not control it to disable buttons when OVF Template folder is selected, so for now this was the best solution i could put to get the issue fixed.

Reply:
On top of this adding extra complexity, this goes against pluggability :disappointed: are you sure we can't do it in a nicer way? What is this thing actually doing?

Reply:
> On top of this adding extra complexity, this goes against pluggability disappointed are you sure we can't do it in a nicer way? What is this thing actually doing?

this should go away with de-explorization, with de-explorization i am trying to use `toolbar :chargeback_rate, :chargeback_rates` approach from controller, so hopefully that will clean the toolbar chooser a little